test/e2e/encryption_at_rest_test.go

```diff
@@ -198,7 +259,7 @@ func checkIfEncryptionsAreDisabled(projectID string) (areEmpty bool, err error)
 	return true, nil
 }
 
-var _ = Describe("Encryption at rest AWS", Label("encryption-at-rest"), func() {
+var _ = Describe("Encryption at rest AWS", Label("encryption-at-rest"), Ordered, func() {
```

Comment:
Did ordering influence flakiness?

Reply:
For this test case yes. We have this `var testData *model.TestDataProvider` global variable that is only set on the spec (`It`). The way Ginkgo behaves might lead to one spec overriding what was set by the other. The proper way to use global vars is to set them using `BeforeEach` so the framework will properly encapsulate it when creating the Test Tree.

Reply:
Making it ordered avoids one spec to override the data set by the other.